client/connector_create_test.go

```diff
@@ -19,24 +19,37 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+var lightRed string = "\033[1;31m"
+var resetColor string = "\033[0m"
+
 func TestConnectorCreateError(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	cli, err := newMockClient("my-namespace", "", "")
-	assert.Assert(t, err)
-
-	_, err = cli.ConnectorCreateFromFile(ctx, "./somefile.yaml", types.ConnectorCreateOptions{
-		Name: "",
-		Cost: 1,
+	var cli *VanClient
+	var err error
+	ns := "namespace-for-testconnectorcreateerror"
+	cli, err = NewClient(ns, "", "")
```

Comment:
Does this need to run against cluster? Either setup mock or cluster client as is done in setupTwoNamespaces...

Reply:
I was going to say 'yes' -- but I just ran full test suite against real-cluster and mock and it looks like the correct answer is 'no'!
This test passed both ways after I put in the code for checking clusterRun -- so I guess we can leave it in there.